travel-journal/index.md

```diff
@@ -0,0 +1,910 @@
+---
+
+## Introduction
+
+In this tutorial, you we’ll build a personal travel journal application using Vue.js and Directus. We’ll go through the process of implementing user authentication with Directus and demonstrate how to interact with a Directus collection using the Vue.js framework.
+
+## Before You Start
+
+You'll need:
+
+- [Node.js](https://nodejs.org/en/) and a code editor installed on your machine.
+- A Directus project created either using [Directus Cloud](https://directus.cloud/) or [run locally on your machine.](https://docs.directus.io/getting-started/quickstart.html). This tutorial on the otherhand uses the Directus SDK via NPM.
+- Some knowledge of the Vue.js Composition API.
+
+## Creating A Vue.js Project
+
+Open up your command prompt or terminal app and run the following command:
+
+```bash
+npm create vue@3
+```
+
+During project creation, when prompted with questions, choose the following configurations:
+
+```bash
+✔ Project name: … ./directus-travel-journal
+✔ Package name: … -directus-travel-journal
+✔ Add TypeScript? … No / Yes
+✔ Add JSX Support? … No / Yes
+✔ Add Vue Router for Single Page Application development? … No / Yes
+✔ Add Pinia for state management? … No / Yes
+✔ Add Vitest for Unit Testing? … No / Yes
+✔ Add an End-to-End Testing Solution? › No
+✔ Add ESLint for code quality? … No / Yes
+```
+
+Once completed, run the following command to install dependencies and start the development server:
+
+```bash
+cd directus-travel-journal
+code .
+npm install
+npm run dev
+```
+
+Once everything is set up, visit the following address  - localhost:5173 **and you should see the vite welcome page:
+
+![Viewing the vite welcome page](./images/1.png)
+
+## Creating a Journal and Users Collection
+
+In your Directus project, navigate to Settings -> Data Model and create a new collection called `journal`. Use `journal` as the table name and leave every other configuration as their defaults.
+
+Create the following fields with the following types:
+
+- `title`: Input - string (*required*).
+- `description`: Textarea - text (*required*).
+- `photo`: Image - uuid (*required*).
+- `city`: Input - string (*required*).
+- `country`: Input - string (*required*).
+
+Add a description of the `journal` collection and proceed to creating the collection.
+
+![Creating the journal collection](./images/2.png)
+
+Head over to Settings→Access Control, create customer role with only App Access. Give it create, read, update and delete access to the `journal` collection, add a description and then proceed to creating the customer role while taking note of the customer role id:
+
+![Creating a customer role](./images/3.png)
+
+After creating the customer role, you will be directed to the Access Control page. Click on the Public role and enable create, read, update and delete access to the `journal` collections and also for `directus_files` under system collections:
+
+![Authorizing public role](./images/4.png)
```

Comment:
```suggestion
In your Directus project, navigate to Settings → Access Control and create a `Customer` role with only App Access. Give it create, read, update and delete access to both the `journal` and `directus_files` collection. Take note of the Customer role ID in the sidebar of the role permissions overview page.

![The access control settings for the public role show read, write, edit, and delete operations for both the journal and Directus files collections](./images/4.png)
```

This was needlessly repetitive - you want them to do the same thing for two collections, so combine them. If users needed help navigating to the `directus_files`, the image acts as the support.